nightmare/paths/metadataprops.spec.js

```diff
@@ -0,0 +1,181 @@
+
+    const numberOfProps = await nightmare.evaluate(
+      selector => document.querySelectorAll(selector).length,
+      localSelectors.templateProperties
+    );
+
+    if (numberOfProps !== 15) {
+      fail('amount of props have change, update this test');
+    }
+  });
+
+  it('should create a template with all the properties', async () => {
+    await nightmare
+      .waitToClick(selectors.settingsView.templatesButton)
+      .waitToClick(selectors.settingsView.addNewTemplate)
+      .write(selectors.settingsView.documentTemplateNameForm, 'All props');
+
+    for (let propIndex = 1; propIndex <= 15; propIndex += 1) {
+      await nightmare.waitToClick(localSelectors.propertiesButtons(propIndex));
+    }
+
+    await nightmare
+      .waitToClick(localSelectors.editRelationshipProperty)
+      .selectByLabel(localSelectors.relationshipPropertyType, 'Perpetrator')
+      .waitToClick(selectors.settingsView.saveTemplateButton)
+      .waitToClick('.alert.alert-success');
+  });
+
+  it('should create an entity filling all the props', async () => {
+    await nightmare
+      .waitToClick(selectors.navigation.uploadsNavButton)
+      .waitToClick(selectors.uploadsView.newEntityButtom)
+      .type(selectors.newEntity.form.title, 'Entity with all props')
+      .selectByLabel(selectors.newEntity.form.type, 'All props')
+      .wait(localSelectors.form.text)
+      .type(localSelectors.form.text, 'demo text')
+      .type(localSelectors.form.numeric, '42')
+      .selectByLabel(localSelectors.form.select, 'This')
+      .click(localSelectors.form.multiselectOptionOne)
+      .click(localSelectors.form.relationshipOptionOne)
+      .selectDate(localSelectors.form.date, '08/09/1966')
+      .selectDate(localSelectors.form.dateRangeFrom, '23/11/1963')
+      .selectDate(localSelectors.form.dateRangeTo, '12/09/1964')
+      .click(localSelectors.form.multidateAddDateButton)
+      .selectDate(localSelectors.form.multidateInputOne, '23/11/1963')
+      .selectDate(localSelectors.form.multidateInputTwo, '12/09/1964')
+      .click(localSelectors.form.multidaterangeAddButton)
+      .selectDate(localSelectors.form.multidaterangeFromInputOne, '23/11/1963')
+      .selectDate(localSelectors.form.multidaterangeToInputOne, '12/09/1964')
+      .selectDate(localSelectors.form.multidaterangeFromInputTwo, '23/11/1963')
+      .selectDate(localSelectors.form.multidaterangeToInputTwo, '12/09/1964')
+      .type(localSelectors.form.richText, '*** smile ***')
+      .type(localSelectors.form.linkLabel, 'Huridocs')
+      .type(localSelectors.form.linkUrl, 'https://www.huridocs.org/')
+      .type(localSelectors.form.image, 'test')
+      .type(localSelectors.form.media, 'test')
+      .insert(localSelectors.form.geolocationLat, '46')
+      .insert(localSelectors.form.geolocationLon, '6')
+      .click(localSelectors.form.save)
+      .waitToClick('.alert.alert-success');
+  });
+
+  it('should be able to remove all the values from properties', async () => {
```

Comment:
@konzz i think this one should actually have a expect testing the values are properly deleted.